### DISCUSSION TITLE
オープンソースに参加する人の文言が、CODEを理解をしたいだけの人には障害があるのかも

### DISCUSSION THREAD

**Discussion by motoi** [copilot-jp/project-sprint#376] [Ideas]
トップページにいきなりでてくるコントリビューターという言葉に割と引っかかる人が多く、それで「自分が読むものじゃないのかな？」とおもってしまうそうな。
参加方法はGithubのほうだけに乗せるとか、そもそもトップページから完全に分離して別のメニューにするとかでもよいかもとおもった。

**Comment by KokoroKagawa** [comment]
トップページ、Project Sprint Guideの↓の記述のことでしょうか。
>フィードバックは、[コントリビューターとしての参加の方法](https://github.com/contributing)のページを参考に、GitHubのIssueでお寄せください。

めも
- 「CODEの理解をしたいだけの人」が疑問・質問やフィードバックを寄せてくださる可能性があるのであれば、どこかしらに導線は残しておきたい。
- コントリビューターという言葉に引っかかって読んでもらえなくなることは回避したいが、How to contributeの前に、What is contributionを定義したほうがよいかも。
